tests/html.rs

```diff
@@ -217,6 +217,52 @@ fn html_test_11() {
     assert_eq!(expected, s);
 }
 
+#[test]
```

Comment:
These are fine tests! I'd only give them more descriptive names, so we can immediately tell that they're testing custom header ids and classes.

Also, could you try getting a little creative with trying to come up with weird edge cases?  What happens when the curly braces are on different lines? What if they are not closed at all? What happens on `{#this.sort.of.stuff}`?

Reply:
Also, we usually add tests for this sort of stuff in `specs`, for which then tests are auto-generated. Not sure if that is the most useful here, but something you may want to consider.